pkg/blackduck/ctl_blackduck.go

```diff
@@ -304,6 +307,14 @@ func (ctl *Ctl) SetFlag(f *pflag.Flag) {
 			for _, pvc := range pvcStructs.Data {
 				ctl.Spec.PVC = append(ctl.Spec.PVC, pvc)
 			}
+		case "postgres-claim-size":
+			for i := range ctl.Spec.PVC {
+				if ctl.Spec.PVC[i].Name == "blackduck-postgres" { // update claim size and return
```

Comment:
@mphammer how the priority will be set, if the claim size is provided using this parameter as well pvc file path?